express-hello-world/server.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res) => {
+  console.log(req.method);
+  res.send('SERVER IS LIVE');
+});
+
+app.listen(8080, () => {
+  console.log('Server is active');
```

Comment:
For future use, it's convenient to log the port you are listening on so you'll know what port to access without looking at the code.